unikernel.ml

```diff
@@ -49,6 +49,41 @@ struct
   module Store = Storage.Make (BLOCK)
   module Map = Map.Make (String)
 
+  let generate_csrf_token ?(update_user = false) ?(user = None) store now =
+    if update_user then
```

Comment:
Could you please describe here in a comment what "update_user" is used for?

Reply:
And `?(user = None)` makes this a "user option option" - is this intended? I.e. as you write below, you need to pass `~user:(Some user)` now... if you would make it `?user` here, you could pass `~user:user` (or even `~user`) below.

Reply:
done in bd73bc7 -- it was the case that a `user` was always provided.

Reply:
ehm, [a142cc0](https://github.com/robur-coop/mollymawk/pull/65/commits/a142cc076e4ea7ed057539c991b85a88ba60931e) is the fixed up commit that actually compiles